api/pkg/apis/v1alpha1/providers/stage/materialize/materialize.go

```diff
@@ -44,6 +45,10 @@ var (
 type MaterializeStageProviderConfig struct {
```

Comment:
We need similar wait option in create stage provider. patch stage provider can only support solution patch, so far this may not need the wait option.